core/node/api_server/src/web3/metrics.rs

```diff
@@ -394,7 +394,9 @@ impl ApiMetrics {
             Web3Error::ProxyError(err) => {
                 tracing::warn!("Error proxying call to main node in method `{method}`: {err}");
             }
-            _ => { /* do nothing */ }
+            _ => {
+                tracing::debug!("Error in method `{method}`: {err:#}");
```

Comment:
okay, fine, I can even revert this change completely if it is better this way. @perekopskiy added it most likely during 
 local debugging and it might be helpful there